app/_data/tables/features/gateway.yml

```
@@ -0,0 +1,128 @@
+columns:
+  - name: Open Source
+    cta: Get Started
+    cta_link: /
+    key: oss
+  - name: Kong Enterprise
+    cta: Contact Sales
+    cta_link: http://konghq.com/contact-sales
+    key: enterprise
+features:
+  - name: API Infrastructure Modernization
+    items:
+      - name: Fast, Lightweight, Cloud-Native API Gateway
+        oss: true
+        enterprise: true
+      - name: End-to-End Automation
+        tooltip: Drive a GitOps flow of API design and execution
+        oss: true
+        enterprise: true
+      - name: Kubernetes Ingress Controller
+        tooltip: Deploy APIs to Kubernetes in a native fashion
+        oss: true
+        enterprise: true
+      - name: Gateway Mocking
+        tooltip: Mock API responses directly on the API gateway
+        oss: false
+        enterprise: true
+      - name: "Kong Manager: Admin GUI"
+        tooltip: Visually manage Kong cluster, plugins, APIs, and consumers
+        oss: false
+        enterprise: true
+      - name: 
+        oss: true
+        enterprise: true
+  - name: Traffic Management and Transformations
+    items:
+      - name: Basic Traffic Control Plugins
+        tooltip: Manage ACME certificates, basic rate limiting, and lightweight caching 
+        oss: true
+        enterprise: true
+      - name: Simple Data Transformations
+        tooltip: Add or remove headers, JSON data, or query strings 
+        oss: true
+        enterprise: true
+      - name: gRPC Transformations
+        tooltip: Translate requests from gRPC-Web and REST to backend gRPC services
+        oss: true
+        enterprise: true
+      - name: GraphQL
+        tooltip: Convert GraphQL queries to REST requests. Rate limit and cache GraphQL queries
```

```suggestion
        tooltip: Convert GraphQL queries to REST requests. Rate limit and cache GraphQL queries.
```
If one has a period, then both should